src/theme/theme.ts

```diff
@@ -37,6 +37,17 @@ const generatePalette = (
 export const palette: PaletteOptions = {
   ...generatePalette(ui, 'ui'),
   ...generatePalette(collage, 'collage'),
+  mode: 'light',
+  primary: { main: '#954661', dark: '#7b1839', light: '#a7647a' },
```

Comment:
We need to know what colors these. 

i would put these colors in a file that looks similar to `colors/ui` or `colors/collage`.

Reply:
+1